flink-connectors/flink-connector-pulsar/src/test/resources/containers/txnStandalone.conf

```diff
@@ -14,8 +14,7 @@
 
 ### --- General broker settings --- ###
 
-# Zookeeper quorum connection string
-zookeeperServers=
+# Zookeeper quorum connection stringzookeeperServers=
```

Comment:
I did not find this change in the original PR, and it looks suspicious.

Reply:
This looks indeed as an unintended change.

Reply:
Yep. This should be a mistake on my side.